DiseasesReimagined/ImaginationLoader.cs

```diff
@@ -73,8 +74,10 @@ public sealed class ImaginationLoader {
 #if DEBUG
             if (loader == null)
                 PUtil.LogWarning("IsFinalDestination() called before save loaded!");
-#endif
-            return loader?.worldGen?.Settings?.world?.name == FINALDEST_KEY;
+#endif 
+            // Not sure what the new API is for this.
+            // return loader?.worldGen?.Settings?.world?.name == FINALDEST_KEY;
+            return false;
```

Comment:
Need to figure out the new way to do this, if still needed.

Reply:
The new merged version of `ImaginationLoader` is available at https://github.com/peterhaneve/ONIMods/blob/main/DecorReimagined/ImaginationLoader.cs.